django_admin_shell/templates/django_admin_shell/shell.html

```diff
@@ -24,6 +24,14 @@ <h5> Django version : {{django_version}} </h5>
         </div>
     </div>
 
+    <script>
+        function save_snippet(code) {
+            var encoded = encodeURIComponent(code);
```

Comment:
It could be danger for large piece of code. 
I recommend to prune code to 2000 chars. 
Please remember that GET payload has some limitations. 
https://stackoverflow.com/questions/2659952/maximum-length-of-http-get-request